### PR TITLE
Sk/test label bot

### PR DESCRIPTION
**Pull request by snopoke**

### CODE REVIEW

**Comment by snopoke** [comment]
@dimagimon retrigger-local review

**Comment by snopoke** [comment]
@dimagimon retrigger review

**Comment by snopoke** [comment]
@dimagimon retrigger-local review

**Comment by snopoke** [comment]
@dimagimon retrigger-local review

**Comment by snopoke** [comment]
@dimagimon retrigger-local review

**Comment by snopoke** [comment]
@dimagimon retrigger-local review

**Comment by snopoke** [comment]
@dimagimon retrigger-local all

**Comment by snopoke** [comment]
@dimagimon retrigger-local auto-labels

File: .github/labels.yml [hunk 1]
```diff
@@ -1,18 +1,20 @@
+disabled_actions:
+  - triage
+  - review
+  - wip
+  - lgtm
```
Comment (millerdev) [inline .github/labels.yml:5]: What does `disabled_actions` do? I couldn't find any documentation on it.

**Review by millerdev — COMMENTED** [review]
What label bot is this configuration for? Is it https://github.com/marketplace/actions/github-labeler?

Edit: found it. https://github.com/dimagi/label-bot

File: .github/labels.yml (hunk 1 above)
Comment (millerdev) [reply .github/labels.yml:5]: https://github.com/dimagi/label-bot#disabling-actions

Comment (snopoke) [reply .github/labels.yml:5]: This just prevents PR's getting status updates for workflows that we aren't using.

**Pull request merged**